boot/src/main/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<configuration>
```

Comment:
Did you consider that this could be handled as an Ops concern, ie, by setting `log-opts` in daemon.json of Docker?

Reading material for ref: https://docs.docker.com/config/containers/logging/configure/